worldedit-core/src/main/java/com/sk89q/worldedit/session/PlacementType.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.session;
+
+import com.sk89q.worldedit.IncompleteRegionException;
+import com.sk89q.worldedit.extension.platform.Actor;
+import com.sk89q.worldedit.extension.platform.Locatable;
+import com.sk89q.worldedit.math.BlockVector3;
+import com.sk89q.worldedit.regions.RegionSelector;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public enum PlacementType {
+    WORLD("worldedit.toggleplace.world", "worldedit.toggleplace.world-offset") {
+        @Override
+        public BlockVector3 getPlacementPosition(RegionSelector selector, Actor actor) throws IncompleteRegionException {
+            return BlockVector3.ZERO;
+        }
+    },
+
+    PLAYER("worldedit.toggleplace.player", "worldedit.toggleplace.player-offset") {
+        @Override
+        public BlockVector3 getPlacementPosition(RegionSelector selector, Actor actor) throws IncompleteRegionException
+        {
```

Comment:
Is this `{` and the one on line 58 intentionally on a newline? It's not tripping checkstyle which confuses me, so is there a specific reason they're here?

Reply:
totally not intentional. might have been autoformat...
I'll fix it and look for other places where that happened in my PRs